src/devices/bus/heathzenith/h8/wh_8_64.cpp

```diff
@@ -0,0 +1,314 @@
+// license:BSD-3-Clause
+// copyright-holders:Mark Garlanger
+/***************************************************************************
+
+  Heathkit WH-8-64 32k/64k Dynamic RAM board
+
+****************************************************************************/
+
+#include "emu.h"
+
+#include "wh_8_64.h"
+
+#include "machine/ram.h"
+
+namespace {
+
+class wh_8_64_device : public device_t, public device_h8bus_card_interface
+{
+public:
+
+	wh_8_64_device(const machine_config &mconfig, const char *tag, device_t *owner, u32 clock = 0);
+
+protected:
+
+	wh_8_64_device(const machine_config &mconfig, device_type type, const char *tag, device_t *owner, u32 clock = 0);
+
+	virtual void device_start() override ATTR_COLD;
+	virtual void device_reset() override ATTR_COLD;
+	virtual void device_add_mconfig(machine_config &config) override ATTR_COLD;
+
+	bool m_installed;
+
+	required_device_array<ram_device, 8> m_ram;
```

Comment:
Since you can’t use `-ramsize` with these anyway, you’re better off just using `memory_share_array_creator` rather than an array of `ram_device`.